blockstack-sdk/src/main/java/org/blockstack/android/sdk/model/GetFileOptions.kt

```diff
@@ -7,12 +7,14 @@ import java.net.URL
  * An object to configure options for `getFile` operations.
  *
  * @property decrypt attempt to decrypt the file
+ * @property verify whether the content should be verified, only to be used when UserSession.putFile was set to sign = true.
```

Comment:
> verify whether the content should be verified, only to be used when UserSession.putFile was set to sign = true.

Indicate if the content should be verified. Verification also requires that UserSession.putFile is set to sign=true.

Reply:
@moxiegirl The content is verified if the content was signed during upload (`putFile`). Therefore, "was set to sign=true" makes more sense then "is".